src/Stuart/Client.php

```diff
@@ -31,6 +33,24 @@ public function validateJob(Job $job)
         }
     }
 
+    /**
+     * @param $city
```

Comment:
If you want to document this method, please can you add it to every method in the class?